src/renderer/components/+namespaces/namespace.store.ts

```diff
@@ -130,41 +100,50 @@ export class NamespaceStore extends KubeObjectStore<Namespace> {
   }
 
   @action
-  setContext(namespace: string | string[]) {
```

Comment:
There are some breaking changes with names of class methods. 
If this is critical we could add wrappers with old names and mark as `@deprecated`. Wdyt?